shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1488,6 +1497,24 @@ private void createAndSendWindowChangeEvent(@NonNull SemanticsNode route) {
         sendAccessibilityEvent(event);
     }
 
+    /**
+     * Factory method that creates a {@link AccessibilityEvent#TYPE_WINDOW_CONTENT_CHANGED} and sends
+     * the event to Android's accessibility system.
+     *
+     * It sets the content change types to {@link AccessibilityEvent#CONTENT_CHANGE_TYPE_SUBTREE}
+     *
+     * The given {@code virtualViewId} should be a {@link SemanticsNode} below
+     * which the content changed.
+     */
+    private void createAndSendWindowContentChangeEvent(int virtualViewId) {
```

Comment:
nit: maybe just name this sendWindowContentChangeEvent?

Reply:
Done, and I renamed the one above too, since it is similar.